core/pref.h

```diff
@@ -206,6 +206,7 @@ struct preferences {
 	bool                    coordinates_traditional;
 	enum unit_system_values unit_system;
 	struct units            units;
+	bool					standard_level_grid;
```

Comment:
there looks to be a white space issue here.
I'm not sure which editor you used, but different editors have different ideas how to deal with tabs and indentation and alignment... the easiest way in my experience is to copy the line above and mimic its behavior (because, sadly, for the variable alignment we are not totally consistent between different files... mostly it's spaces, but sometimes there are tabs)
And as I said in the comment on `pref.c` - I'd prefer a different name for this member.

Reply:
I can see, you use tabs for indentation, that's why I left tabs in settings. Apparently that also caused conversion of spaces into tabs. 
My bad. I usually try to mimic convention of the file I'm editing.

Reply:
Also, it's in the wrong section. It used to be in Units initially, and then I moved it into Display.